fffw/encoding/outputs.py

```diff
@@ -93,6 +93,17 @@ def check_buffering(self) -> Optional[List[str]]:
             prev = scene
         return meta.streams
 
+    @property
+    def meta(self) -> Optional[Meta]:
```

Comment:
base `transform` method for filters is defined in `Node` class. For codecs corresponding base class is `Dest`, there things should be placed there.

Reply:
Moved `transform` and `meta` to `Dest` class.